src/options.ts

```diff
@@ -326,16 +372,15 @@ export type getColumn = {transform: () => ValueArray; label?: string};
 type setColumn = <T extends ValueArray | Float32Array | Float64Array>(v: T) => T;
 
 // Like column, but allows the source to be null.
-export function maybeColumn(source: ValueAccessor | ((data: DataArray) => void)): [getColumn, setColumn] | [null?] {
+export function maybeColumn<T extends Datum>(
+  source: ValueAccessor<T> | ((data: DataArray) => void)
+): [getColumn, setColumn] | [null?] {
   return source == null ? [source] : column(source);
 }
 
-export function labelof(value: any, defaultValue?: string) {
-  return typeof value === "string"
-    ? value
-    : value && (value as {label?: string}).label !== undefined
-    ? (value as {label?: string}).label
-    : defaultValue;
+type MaybeLabel = string | {label: string} | null | undefined;
+export function labelof(value: MaybeLabel, defaultValue?: string) {
```

Comment:
labelof must accept any ValueAccessor; it just checks if there is a label property (which the user might assign on a typed array if they want!)